vnext/local-cli/generator-windows/templates/cpp/src/MainPage.xaml

```diff
@@ -1,13 +1,13 @@
-﻿<Page
-    x:Class="<%=name%>.MainPage"
+﻿<Page 
```

Comment:
it's possible to unify this xaml with the cs xaml?

Reply:
Agree, but probably out of scope of this PR. Filed issue: #5128